docs/source/api.rst

```diff
@@ -1,12 +1,8 @@
 API
 ===
 
-.. attention::
-   This library is currently in the tech preview stage - it is possible that models 
-   and their APIs will change in incompatible ways before ``1.0.0``.
-
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 3
```

Comment:
Too messy. A ToC should provide a quick overview of the structure of the documentation, 3 is very overwhelming.

Reply:
Done.